code/workspaces/common/src/config/catalogue_config.json

```diff
@@ -1,3 +1,17 @@
 {
-  "available": true
+  "available": true,
```

Comment:
It looks like this file is now covering both the catalogue and the repo, which I've been viewing as separate things - you could have a central asset repo but no catalogue.  I don't want to see a proliferation of configmap files, perhaps we could split this file up into
`
{
  "catalogue": {
    "available": true,
  },
  "repo:": {
    "types": {
      etc
    }
  }
}
`

Reply:
This sounds like a good plan to me; I'll make the change!